docs/content/what-is-temporal.md

```diff
@@ -8,7 +8,10 @@ tags:
 
 import CenteredImage from "../components/CenteredImage.js"
 
-Temporal is a scalable and reliable runtime for Reentrant Processes called Temporal Workflow Executions.
+<!-- prettier-ignore -->
+import * as WhatIsAWorkflowExecution from '../content/what-is-a-workflow-execution.md'
+
+Temporal is a scalable and reliable runtime for Reentrant Processes called <preview page={WhatIsAWorkflowExecution}>Temporal Workflow Executions</preview>.
 
 :::note Temporal's tenth rule
```

Comment:
Can we link somewhere, maybe "tenth rule" to the origin? So that people unfamiliar get the joke / aren't wondering what the first 9 rules are. https://en.wikipedia.org/wiki/Greenspun%27s_tenth_rule

Reply:
Sure!